css/styles.css

```diff
@@ -102,6 +103,15 @@ html {
   margin: 12px 0;
 }
 
+#validation-msg {
```

Comment:
- [ ] On your readme, kindly add the missing sections according to the [Microverse Readme Template](https://github.com/microverseinc/readme-template).  

  - Please, include the missing Acknowledgments section.
  - Please, include the missing Show your support section.
  - Please, include the missing Contributing section.
  - Please, include the missing License section.
 
## Explanation

Kindly remember that the Microverse Readme Template is required in almost all projects.   All sections are required except for the "Tests" and "Live demo" if not required by the project.

## Hints

Don't worry about having to make a huge readme on each project.   You can just do it once, and use it for all your projects (just customize the sections for each project).  

That way, your project will be well documented 💪